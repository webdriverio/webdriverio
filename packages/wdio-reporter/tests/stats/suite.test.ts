import SuiteStats from '../../src/stats/suite'

test('should get initialised', () => {
    const suite = new SuiteStats({
        cid: '0-0',
        title: 'foobar',
        fullTitle: 'barfoo',
        description: 'some description',
        tags: ['foo', 'bar']
    })

    delete suite.start
    expect(suite).toMatchSnapshot()
})
