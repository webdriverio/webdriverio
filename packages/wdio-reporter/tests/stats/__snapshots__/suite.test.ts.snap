// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get initialised 1`] = `
Object {
  "_duration": 0,
  "cid": "0-0",
  "description": "some description",
  "fullTitle": "barfoo",
  "hooks": Array [],
  "hooksAndTests": Array [],
  "suites": Array [],
  "tags": Array [
    "foo",
    "bar",
  ],
  "tests": Array [],
  "title": "foobar",
  "type": "suite",
  "uid": "foobar",
}
`;
